lively-server/index.js

```diff
@@ -0,0 +1,48 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import { graphqlExpress, graphiqlExpress } from 'apollo-server-express';
+import { makeExecutableSchema } from 'graphql-tools';
+import models from './models';
+import path from 'path';
+import { fileLoader, mergeTypes, mergeResolvers } from 'merge-graphql-schemas';
+import seed from './seed';
+import cors from 'cors';
+import { addUser } from './auth';
+const typeDefs = mergeTypes(fileLoader(path.join(__dirname, './schema')));
+const resolvers = mergeResolvers(fileLoader(path.join(__dirname, './resolvers')));
+
+export const schema = makeExecutableSchema({
+    typeDefs,
+    resolvers,
+    printErrors: true,
+});
+
+// const PORT = process.env.PORT || 8081;
+const PORT = 8081;
+const SECRET = 'qwertyuiopljhgfdsaasdfghjkl';
+
+const app = express();
+app.use(cors('*'));
```

Comment:
don't ever, ever, ever, ever, EVER do this. I'm assuming you're aware of what CORS is, try to think about why this isn't an ok thing to do